### PR TITLE
Document mvn cq:await-release -Dcq.version=$VERSION

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: integration-test-groups/aws2/README.adoc [hunk 1]
```diff
@@ -1,10 +1,8 @@
-== AWS 2 isolated integration tests
+= AWS 2 isolated integration tests
 
 === Localstack
 
-The tests not impacted by https://github.com/apache/camel-quarkus/issues/2216 are run against Localstack by default.
-
-The tests impacted by https://github.com/apache/camel-quarkus/issues/2216 are ignored by default.
+The tests able to run against Localstack are run agains Localstack by default.
```
Comment (VratislavHais) [inline integration-test-groups/aws2/README.adoc:5]: t in second against missing

**Comment by VratislavHais** [comment]
I can confirm that at least aws2-sqs is ran using localstack by default (and completes successfully).

**Review by aldettinger — APPROVED** [review]
This await-release mojo seems to be really useful :+1:

File: integration-test-groups/aws2/README.adoc (hunk 1 above)
Comment (ppalaga) [reply integration-test-groups/aws2/README.adoc]: Thanks, fixed in 69a57ea

**Pull request merged**